unsplash/photo.go

```diff
@@ -38,13 +38,23 @@ type ExifData struct {
 	Iso          *int    `json:"iso"`
 }
 
+// Tag lists can be applied to any photo
+type Tag struct {
+	Type  *string `json:"type"`
+	Title *string `json:"title"`
+}
+
 // Photo represents a photo on unsplash.com
 type Photo struct {
 	ID           *string    `json:"id"`
 	CreatedAt    *time.Time `json:"created_at"`
+	UpdatedAt    *time.Time `json:"updated_at"`
 	Width        *int       `json:"width"`
 	Height       *int       `json:"height"`
 	Color        *string    `json:"color"`
+	Description  *string    `json:"description"`
+	AltText      *string    `json:"alt_description"`
```

Comment:
Let's rename this to `AltDescription` to keep it same as other fields.

Reply:
Done.